save-orchestrator-common/src/main/kotlin/com/saveourtool/save/orchestrator/service/HeartBeatInspector.kt

```diff
@@ -25,7 +24,6 @@ typealias AgentStateWithTimeStamp = Pair<String, Instant>
  * TODO: can be used to store data about existing agents on orchestrator startup ([#11](https://github.com/saveourtool/save-cloud/issues/11))
  */
 @Component
-@PropertySource("classpath:application.properties")
```

Comment:
Worked without it?
How crone task `run()` in this class will determine necessary property?

Reply:
This cron task is part of spring boot application. All properties will be loaded by Spring Configuration